mantidproject/mantid
# Add DiffSphere support to IDA ConvFit

Fixes [#9198](http://trac.mantidproject.org/mantid/ticket/9198).

To test:
- Load sample: `irs26176_graphite002_red.nxs`
- Load resolution: `irs26173_graphite002_res.nxs`
- Select Fit Type: either Diffusion Sphere or Diffusion Circle
- Select Background: Fit Linear (not essential but gives better fit)
- See that selecting different functions changes the Plot Output options (this plots the fit parameters)
- See that the fit is reasonable (use Delta Function to fit high Q)
